etl/steps/data/garden/worldbank_wdi/2024-05-20/wdi.py

```diff
@@ -77,6 +83,73 @@ def run(dest_dir: str) -> None:
     log.info("wdi.end")
 
 
+def add_regions_to_remittance_data(tb: Table, ds_regions: Dataset, ds_income_groups: Dataset) -> Table:
+    """
+    Add regions to remittance data.
+    """
+
+    tb = tb.reset_index()
+
+    # create a copy so other indicators are not affected
+    regions_tb = tb.copy()
+
+    # create new columns for total remittances (only for countries where remittance cost is available)
+    regions_tb["total_received_remittances"] = regions_tb["bx_trf_pwkr_cd_dt"].where(
```

Comment:
Not sure why this step is needed. Why not just multiplying si_rmt_cost_ib_zs * bx_trf_pwkr_cd_dt and si_rmt_cost_ob_zs * bm_trf_pwkr_cd_dt direclty? The nulls would be generated automatically

Reply:
I need this column to calculate "total remittance volume covered by cost data" to calculate the right average and (now) cover the coverage of the regions.